docs/kb/semgrep-code/reduce-false-positives.md

```diff
@@ -0,0 +1,110 @@
+import DefCrossFile from "/src/components/concept/_def-cross-file.mdx"
+import DefCrossFunction from "/src/components/concept/_def-cross-function.mdx"
+
+# Reduce false positives in `semgrep scan`
+
+The `semgrep scan` command can be used to quickly perform SAST scans. However, you may encounter false positives as you work through your findings. This document presents different strategies to reduce false positives and increase true positives in your scans.
+
+## Customize your rules
+
+If you notice that a specific Semgrep Community rule generates a high rate of false positives, the rule is said to be **noisy**. You can:
+
+- Fork and customize that rule to improve its performance
+- Remove the rule from the scan
+
+### Set up local rules
+
+To have more granular control over the rules in a ruleset, you must add the ruleset to your machine, then configure Semgrep to use those local rules.
+
+1. Navigate to the [<i class="fas fa-external-link fa-xs"></i> Semgrep community rules repository](https://github.com/semgrep/semgrep-rules).
+1. Fork or clone the repository to create a local copy of all the rules.
+  1. To clone, click **Code** then copy and run the cloning command in your CLI. This creates a `semgrep-rules` repository.
+  1. To fork, click **Fork** and follow the steps provided by GitHub. You must also clone the forked repository to your machine.
+1. In your CLI, navigate to your `semgrep-rules` repository.
+1. Find and copy the rules you want to use in a folder within your target codebase. Give the folder a descriptive name, such as `semgrep-rules`.
+1. To use the local rules, run the following command:
+<pre>semgrep scan --config='<span class="placeholder">SEMGREP_RULES_FOLDER/'</span></pre>
+
+### Customize a rule from a Semgrep Community ruleset
+
+1. Edit the noisy rule to improve its performance.
+1. Test your rule improvements by entering:
+<pre>semgrep scan --config='<span class="placeholder">SEMGREP_RULES_FOLDER</span>/<span class="placeholder">NAME_OF_IMPROVED_RULE</span>.yaml'</pre>
+
+### Remove the rule from the scan
+
+Delete the rule from the folder containing your Semgrep rules.
+
+## Use advanced analyses and Pro rules
+
+Optimizing rules can be a time-consuming process. Often, rules are not necessarily noisy, but lack additional analysis to detect true positives while ignoring false positives.
+
+[Semgrep Code](/semgrep-code/overview/) provides cross-function (interprocedural) and cross-file (interfile) analyses. These analyses both reduce false positives and detect true positives that Semgrep OSS can't find.
+
+For some languages and frameworks, such as Java or the Python Django framework, Semgrep also provides advanced analyses that take into account the language's characteristics, framework-specific dataflows, and the like. These analyses are available by default.
+
+To enable these analyses, you must create and sign in to a Semgrep account.
```

Comment:
This reads a bit like it contradicts the sentence immediately before it. What about something like "These analyses are available by default once you've signed in to Semgrep"?